interface/billing/sl_eob_search.php

```diff
@@ -256,8 +259,12 @@ function upload_file_to_client_pdf($file_to_send, $aPatFirstName = '', $aPatID =
   //
 if (($_POST['form_print'] || $_POST['form_download'] || $_POST['form_pdf']) || $_POST['form_portalnotify'] && $_POST['form_cb']) {
   $fhprint = fopen($STMT_TEMP_FILE, 'w');
+  $sqlBindArray = array();
   $where = "";
-  foreach ($_POST['form_cb'] as $key => $value) $where .= " OR f.id = $key";
+  foreach ($_POST['form_cb'] as $key => $value) {
```

Comment:
Need to revert to  - 

foreach ($_POST['form_cb'] as $key => $value) $where .= " OR f.id = $key";

otherwise, the code is not working on testing